config/envs/default.py

```diff
@@ -141,12 +141,12 @@ class DefaultConfig:
 
     TALISMAN_SETTINGS = CommonConfig.TALISMAN_SETTINGS
 
-    COF_FUND_ID = CommonConfig.COF_FUND_ID
-    COF_ROUND2_ID = CommonConfig.COF_ROUND_2_ID
-    COF_ROUND2_W3_ID = CommonConfig.COF_ROUND_2_W3_ID
+    COF_FUND_ID = "47aef2f5-3fcb-4d45-acb5-f0152b5f03c4"
```

Comment:
Why are these being added here? Why not update https://github.com/communitiesuk/funding-service-design-utils/blob/main/fsd_utils/config/commonconfig.py?

Reply:
All fund info will be in the fund store, this is a bit of a shortcut whilst that change goes in but in the long term we shouldn't be hardcoding funds at all (hence a call to COF_FUND_ID will disappear all together)